google.json

```diff
@@ -816,8 +1000,8 @@
     {
         "id": "com.google.android.ext.shared",
         "label": "Android Shared Library",
-        "description": "Used to share common code between apps. It's empty, so this package is useless?",
-        "removal": "delete"
+        "description": "Maybe people thinking its safe to remove or its useless because its empty.\nbut it cause bootloop on my miui 14.",
```

Comment:
Bootloop warning should be placed in the `warning` section. You're indeed correct. Android shared libraries are essential items.